holidays/calendars/thai.py

```
@@ -264,34 +266,37 @@ def __init__(self, calendar=THAI_CALENDAR) -> None:
 
     @staticmethod
     def __is_khmer_calendar(calendar):
-        """
-        Return True if `calendar` is Khmer calendar.
-        Return False otherwise.
+        """Check if the given calendar is the Khmer calendar.
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)